src/Symfony/Component/Debug/Exception/FlattenException.php

```diff
@@ -142,6 +142,12 @@ public function getMessage()
 
     public function setMessage($message)
     {
+        if (false !== strpos($message, "class@anonymous\0")) {
+            $message = preg_replace_callback('/class@anonymous\x00.*?\.php0x?[0-9a-fA-F]++/', function ($m) {
+                return \class_exists($m[0], false) ? get_parent_class($m[0]).'@anonymous' : $m[0];
```

Comment:
technically this can occur for anon. clases without a parent class.. do we care? Same for `setClass`.

Reply:
yes we do, and this is handled: the result will be just `@anonymous`, as the VarDumper component already does.